### PR TITLE
Include name of missing OPM restart file in error.

### PR DESCRIPTION
**Pull request by blattms**
Otherwise it is hard to tell for the (newbie?) user what went wrong.

### CODE REVIEW

**Comment by blattms** [comment]
jenkins build this please

File: opm/simulators/flow/SimulatorFullyImplicitBlackoilEbos.hpp [hunk 1]
```diff
@@ -241,7 +241,7 @@ class SimulatorFullyImplicitBlackoilEbos
                 path.replace_filename(ioconfig.getBaseName() + ".OPMRST");
                 loadFile_ = path;
                 if (!std::filesystem::exists(loadFile_)) {
-                    OPM_THROW(std::runtime_error, "Error locating serialized restart file");
+                    OPM_THROW(std::runtime_error, "Error locating serialized restart file " + saveFile_);
```
Comment (atgeirr) [inline opm/simulators/flow/SimulatorFullyImplicitBlackoilEbos.hpp:244]: Should be loadFile_ I think? Otherwise a good idea.

Comment (blattms) [reply opm/simulators/flow/SimulatorFullyImplicitBlackoilEbos.hpp]: Indeed. fixed.

**Comment by blattms** [comment]
jenkins build this please

**Comment by atgeirr** [comment]
Thanks, merging!

**Pull request merged**